lib/util/platform.js

```diff
@@ -617,21 +617,26 @@ shaka.util.Platform = class {
       const hasCanDisplayType = window.cast && cast.__platform__ &&
           cast.__platform__.canDisplayType;
 
-      // Most Chromecasts can do 1080p.
-      maxResolution.width = 1920;
-      maxResolution.height = 1080;
+      // Some hub devices can only do 720p.  Default to that.
+      maxResolution.width = 1280;
```

Comment:
Just a heads up. The physical resolution of [Google Nest Hub](https://store.google.com/product/nest_hub_2nd_gen) is 1024 x 600. But I guess it can play 720p.

Reply:
It can decode 720p, and with the typical encoding ladder people use, you'd hate to have them play 480p instead.